redhat-cop/casl-ansible
# DNS server should NOT run docker

#### What does this PR do?
Updated groups list to avoid installing OpenShift requirements on the nodes not hosting OpenShift - i.e.: DNS server.

#### How should this be manually tested?
Provision the environment and check that it's functional. Also check that services, e.g.: docker, aren't running on the DNS server.

#### Is there a relevant Issue open for this?
N/A

#### Who would you like to review this?
cc: @redhat-cop/casl @etsauer
